Source/Scene/ModelExperimental/ModelExperimentalSceneMeshPrimitive.js

```diff
@@ -0,0 +1,18 @@
+import GeometryPipelineStage from "./GeometryPipelineStage.js";
+
+export default function ModelExperimentalSceneMeshPrimitive(options) {
+  /**
+   * @type {ModelComponents.Primitive}
+   */
+  this._primitive = options.primitive;
```

Comment:
We talked about this in slack. We use this class like a simple struct, so I'll update these variables to not use underscores and document the fields properly

Reply:
I made this change earlier, though I still need to try this out in a sandcastle to make sure I updated other places properly.